{
  "name": "resumabletcf",
  "version": "1.1.0",
  "description": "A package that let recover failed computations inside try blocks",
  "main": "dist/index.js",
  "files": [
    "/dist"
  ],
  "scripts": {
    "test": "jest --coverage",
    "build": "rollup -c",
    "watch": "rollup -cw"
  },
  "keywords": [
    "try",
    "recover",
    "resumable",
    "catch",
    "javascript"
  ],
  "repository": {
    "type": "git",
    "url": "git+https://github.com/jfet97/resumabletcf.git"
  },
  "author": "Andrea Simone Costa aka jfet97 <andrysimo1997@gmail.com>",
  "license": "MIT",
  "engines": {
    "node": ">=10.0.0"
  },
  "devDependencies": {
    "@babel/core": "^7.5.5",
    "@babel/preset-env": "^7.5.5",
    "babel-jest": "^24.8.0",
    "jest": "^24.8.0",
    "rollup": "^1.18.0",
    "rollup-plugin-babel-minify": "^9.0.0",
    "rollup-plugin-cleanup": "^3.1.1"
  },
  "jest": {
    "transform": {
      "^.+\\.js$": "babel-jest"
    }
  }
}
